src/pages/dashboard/hackathon/[listing]/submissions.tsx

```diff
@@ -114,6 +114,8 @@ function BountySubmissions({ listing }: Props) {
               totalPaymentsMade={totalPaymentsMade}
               rewards={Object.keys(bounty?.rewards || {})}
               bountyId={bounty?.id}
+              bounty={bounty}
```

Comment:
why are you passing bounty and submissions objects?
is there a lot of props that you need from these objects?